src/main/java/uk/ac/cam/cl/dtg/segue/dao/kafkaStreams/UserStatisticsStreamsApplication.java

```diff
@@ -572,4 +590,17 @@ private static Calendar roundDownToDay(Calendar calendar) {
         return calendar;
     }
 
+
+    /**
+     * Method to expose streams app details and status
+     * @return map of streams app properties
+     */
+    public static Map<String, Object> getAppStatus() {
```

Comment:
There is a fair bit of code which could be shared between UserStatisticsStreamsApplicaiton and SiteStatisticsStreamsApplication. We could remove some duplication through inheritance or composition etc.

Reply:
I had considered using some form of inheritance in future, for example both apps use almost identical configs. Will look into it in future

Reply:
Yep cool, I agree it doesn't need to be done now. Maybe a TODO Refactor comment would help remind us if we get around to cleaning things up.